src/app/bav/steps.js

```diff
@@ -28,4 +27,32 @@ module.exports = {
     controller: confirmDetails,
     next: APP.PATHS.DONE,
   },
+  [`${APP.PATHS.CANNOT_PROCEED}`]: {
+    controller: escapeJourney,
```

Comment:
I think typically we align the path name with the controller/field name so that it's easy to find the files you're looking for. What do you think about doing that here?

Reply:
I'll rename all instances of escapeJourney cannotProceed